src/devices/bus/hp_hil/hlekbd.cpp

```diff
@@ -284,10 +284,8 @@ bool hle_device_base::hil_write(uint16_t *pdata)
 	uint8_t data = *pdata & 0xff;
 	bool command = BIT(*pdata, 11);
 
-	if (LOG) {
-		logerror("rx from mlc %04X (%s addr %d, data %02X)\n", *pdata, \
-		 command ? "command" : "data", addr, data);
-	}
+	LOG("rx from mlc %04X (%s addr %d, data %02X)\n", *pdata, \
+	 command ? "command" : "data", addr, data);
```

Comment:
There's no need to use a backslash line continuation here - C++ is largely insensitive to whitespace.  Also, please indent statement continuation by two levels to make it obvious it's statement continuation and not a new scope block.